packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -197,9 +197,15 @@ const pushTransaction = async (request: Request<MessageTypes.PushTransaction>) =
             );
         }
         if (isSolanaError(error)) {
-            throw new Error(
-                `Solana error code: ${error.context.__code}. Please try again or contact support.`,
-            );
+            if (error.context.__code === -32002) {
```

Comment:
This situation is a bit more complicated, since there can be different causes behind the preflight error, eg. compute limit, not enough funds, instruction problems...

I think this would be the correct condition: 
```
isSolanaError(error,  SOLANA_ERROR__JSON_RPC__SERVER_ERROR_SEND_TRANSACTION_PREFLIGHT_FAILURE) &&
isSolanaError(error.cause, SOLANA_ERROR__TRANSACTION_ERROR__BLOCKHASH_NOT_FOUND)
```